pkgs/data/themes/orchis/default.nix

```
@@ -0,0 +1,35 @@
+{ accentColor ? "blue"
+, lib, stdenv, fetchFromGitHub, gtk3, gnome-themes-extra, gtk-engine-murrine }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, gtk3, gnome-themes-extra, gtk-engine-murrine
, accentColor ? "blue" }:
```

options are usually at the end of inputs.